examples/typescript/evm/account.transfer.ts

```diff
@@ -48,14 +48,18 @@ console.log(
   `Sending 0.01 USDC from ${sender.address} to ${receiver.address}...`
 );
 
-const { status, transactionHash } = await sender.transfer({
+const transfer = await sender.transfer({
   to: receiver,
   amount: "0.01",
   token: "usdc",
   network: "base-sepolia",
 });
 
-console.log(`Transfer status: ${status}`);
+const finalizedTransfer = await transfer.waitForResult({
```

Comment:
@rohan-agarwal-coinbase @yuga-cb @alex-stone now that we've had multiple discussions around this DevX, I've come full circle and am leaning towards this API, especially as we've been adding more Actions onto Accounts to make them easy to use. It feels more natural than a `sender.waitForTransferResult`, or pulling `waitForTransferResult` as some standalone function. Thoughts?